docs/components/modeler/bpmn/inclusive-gateways/inclusive-gateways.md

```diff
@@ -0,0 +1,97 @@
+---
+id: inclusive-gateways
+title: "Inclusive gateway"
+description: "An inclusive gateway (or OR-gateway) allows you to make a decision based on data."
+---
+
+:::note
+
+Currently, we've add support for the diverging (i.e. splitting, forking) inclusive gateway, not yet add support for the converging (i.e. merging, joining) inclusive gateway.
+
+- In addition, a combination of parallel and exclusive gateways can be used to merge the flows again.
+
+:::
+
+The inclusive gateway (or OR-gateway) allows for making multiple decisions, not just one.
+
+![process](assets/inclusive-gateway.png)
+
+If an inclusive gateway has multiple outgoing sequence flows, all sequence flows must have a `conditionExpression` to define when the flow is taken. The gateway can have one sequence flow be defined as the default flow.
+
+When an inclusive gateway is entered, the `conditionExpression` is evaluated. The process instance takes all sequence flows where the condition is fulfilled.
+
+For example: Cources selected include `pasta` and `salad`
+
+![process](assets/inclusive-gateway-1.png)
+
+For example: Cources selected include `pasta`, `steak` and `salad`
+
+![process](assets/inclusive-gateway-3.png)
+
+For example: Cources selected include `steak` and `salad`
+
+![process](assets/inclusive-gateway-4.png)
```

Comment:
🔧 IMO the behavior is already clear with the other examples. I think we can remove this one, but it's fine to keep it. I'll leave it up to you.